plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -99,6 +102,19 @@ export class GithubPublisher implements PublisherBase {
       /\.git$/,
       '/blob/master/catalog-info.yaml',
     );
+
+    try {
+      await enableBranchProtectionOnDefaultRepoBranch({
+        owner,
+        client,
+        repoName: name,
+      });
+    } catch (e) {
+      throw new Error(
+        `Failed to add branch protection to '${name}': ${e.message}`,
```

Comment:
We typically do `Failed to x, ${e}` instead. Using the entire error so that we include the name to see what type of error it is, e.g. `'NotFoundError'`, and using comma to avoid `Error: Failed to X: Error: Failed to Y`.